File: Python/C/FizzBuzz.py
# 入力を受け取る
N = int(input())

# 1 から N を for 文でループさせる
for i in range(1, N + 1):
    if i % 3 == 0 and i % 5 == 0:
        # 3 でも 5 でも割り切れる場合
        print("Fizz Buzz")
    elif i % 3 == 0:
        # 3 で割り切れる場合
        print("Fizz")
    elif i % 5 == 0:
        # 5 で割り切れる場合
        print("Buzz")
    else:
        # いずれにも該当しない場合は数字を出力する
        print(i)

File: C++/C/FizzBuzz.cpp
#include <iostream>

using namespace std;

int main() {
    // 入力を受け取る
    int N;
    cin >> N;

    // 1 から N を for 文でループさせる
    for (int i = 1; i <= N; ++i) {
        if (i % 3 == 0 && i % 5 == 0) {
            // 3 でも 5 でも割り切れる場合
            cout << "Fizz Buzz" << endl;
        } else if (i % 3 == 0) {
            // 3 で割り切れる場合
            cout << "Fizz" << endl;
        } else if (i % 5 == 0) {
            // 5 で割り切れる場合
            cout << "Buzz" << endl;
        } else {
            // いずれにも該当しない場合は数字を出力する
            cout << i << endl;
        }
    }

    return 0;
}